core/frontend/src/test/QuantityFormatter.test.ts

```diff
@@ -79,6 +79,20 @@ describe("Quantity formatter", async () => {
     expect(actual).toBe(expected);
   });
 
+  it("Length default parser should handle format", async () => {
+    const numericVal = 6.2484; // 20'-6" in meters
+
+    await quantityFormatter.setActiveUnitSystem("imperial");
+    expect(quantityFormatter.activeUnitSystem).toBe("imperial");
+    const imperialParserSpec = await quantityFormatter.getParserSpecByQuantityType(QuantityType.Length);
+    const imperialFormatSpec = await quantityFormatter.getFormatterSpecByQuantityType(QuantityType.Length);
+    const stringVal = quantityFormatter.formatQuantity(numericVal, imperialFormatSpec);
+    expect(stringVal).toBe(`20'-6"`);
+    const parsedVal = quantityFormatter.parseToQuantityValue(`20'-6"`, imperialParserSpec);
```

Comment:
While this works, I feel that it would be better to use `stringVal` instead of the hard-coded string. The previous line already makes sure that it has the expected value, and passing `stringValue` directly reiterates that the parser can correctly parse the output from the formatter.

```suggestion
    const parsedVal = quantityFormatter.parseToQuantityValue(stringVal, imperialParserSpec);
```

Reply:
gotcha, updated